src/main/java/org/folio/spring/utils/ResourceUtil.java

```diff
@@ -0,0 +1,15 @@
+package org.folio.spring.utils;
+
+import java.nio.file.Files;
+import java.util.List;
+import lombok.SneakyThrows;
+import lombok.experimental.UtilityClass;
+import org.springframework.util.ResourceUtils;
+
+@UtilityClass
+public class ResourceUtil {
```

Comment:
Are you sure, that you need this separate utils class for this metod?

Reply:
I just think that it should be useful for other modules, but I'm fine to remove it.